src/encoding.md

```diff
@@ -344,6 +345,56 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-invalid-csv]: #ex-invalid-csv
+<a name="ex-invalid-csv"></a>
+## Handle invalid CSV data with Serde
+
+[![csv-badge]][csv] [![cat-encoding-badge]][cat-encoding]
+
+We get deserialization errors if invalid CSV data like empty field
+is encountered. Serde converts any such errors to a None value, when applied
+to this Option field.
+
+
+```rust
+extern crate csv;
+extern crate serde;
+#[macro_use]
+extern crate serde_derive;
+
+use std::error::Error;
+use std::process;
+
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "PascalCase")]
```

Comment:
If this serde macro is important then please mention it in description and add link to docs